pages/radar.md

```diff
@@ -7,6 +7,18 @@ permalink: /radar.html
+
+## [Huawei Release Their First RISC-V Development System – Hi3861](https://www.electropages.com/blog/2021/06/huawei-release-their-first-risc-v-development-system-hi3861)
+16-August-2021: Open software has many obvious advantages over proprietary. In general, a free license allows third-party developers - individual programmers and companies - to make their valuable contribution to the development of a product, and consequently, the result obtained has a chance to be more functional and competitive than the same proprietary one. However, open and free can be not only software but also hardware, such as processor architecture. If the [specification](https://riscv.org/technical/specifications/) of such an architecture is available for non-paid and free use, including commercial production of physical chips, such an architecture can as a result have [considerable support](https://riscv.org/members/) from end-producers and programmers, even if its current stage even lags behind the available proprietary solutions. We believe that using open technologies will lead to a more productive result than the development of own or the licensing of third-party proprietary hardware. Not to mention that such an approach is much less affected by the negative impact of unexpected cases such as international sanctions.
+
+## [GitHub’s Engineering Team has moved to Codespaces](https://github.blog/2021-08-11-githubs-engineering-team-moved-codespaces/)
+11-August-2021: Improving software quality (this is what we are aiming for at the Code Quality Foundation) is achieved, among other things, by using various tools such as continuous integration, code checkers, automatic testing utilities, and so on. Configuring and supporting such tools for large projects can obviously take significant resources, including money. What is only worth buying and maintenance servers! The use of cloud technologies, where all the necessary tools are already configured and provided to developers (even directly from an IDE) - an obvious path of least resistance for the project manager. Therefore, we need to focus on developing tools that could work with cloud technologies, such as plugins to find and fix defects in the source code.
```

Comment:
Improving software quality is a key goal of the Code Quality Foundation, which is achieved, among other things, by using various tools, such as continuous integration, code checkers, automatic testing utilities, and so on.  Configuring and supporting such tools for large projects can take significant resources (including money). Purchasing and maintaining servers gets quite expensive! The use of cloud technologies, where all the necessary tools are already configured and provided to developers (even directly from an IDE), is an obvious path of least resistance for the project manager. Therefore, we need to focus on developing tools that could work with cloud technologies, such as plugins, to find and fix defects in the source code.